homeassistant/components/obihai/sensor.py

```diff
@@ -93,86 +93,65 @@ class ObihaiServiceSensors(SensorEntity):
     def __init__(self, pyobihai: PyObihai, serial: str, service_name: str) -> None:
         """Initialize monitor sensor."""
         self._service_name = service_name
-        self._state = None
-        self._name = f"{OBIHAI} {self._service_name}"
+        self._attr_native_value = None
+        self._attr_name = f"{OBIHAI} {self._service_name}"
         self._pyobihai = pyobihai
-        self._unique_id = f"{serial}-{self._service_name}"
-
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return self._name
-
-    @property
-    def native_value(self):
-        """Return the state of the sensor."""
-        return self._state
-
-    @property
-    def available(self):
-        """Return if sensor is available."""
-        if self._state is not None:
-            return True
-        return False
-
-    @property
-    def unique_id(self):
-        """Return the unique ID."""
-        return self._unique_id
-
-    @property
-    def device_class(self):
-        """Return the device class for uptime sensor."""
-        if self._service_name == "Last Reboot":
-            return SensorDeviceClass.TIMESTAMP
-        return None
+        self._attr_unique_id = f"{serial}-{self._service_name}"
+        self._attr_device_class = (
+            SensorDeviceClass.TIMESTAMP if self._service_name == "Last Reboot" else None
+        )
```

Comment:
None is the default:
```suggestion
        if self._service_name == "Last Reboot":
            self._attr_device_class = SensorDeviceClass.TIMESTAMP
```